### ISSUE TITLE
On ARM machines, test_no_numpy_warnings fails

### ISSUE THREAD

**Issue by olebole** [astrofrog/wcsaxes#181]
When running the tests on Debian/arm, the `test_no_numpy_warnings` [fails](https://reproducible.debian.net/rb-pkg/unstable/armhf/wcsaxes.html):
```
____________________________ test_no_numpy_warnings ____________________________

    def test_no_numpy_warnings():
        [...]
    
>       assert len(ws) == 0
E       assert 16 == 0
E        +  where 16 = len([<warnings.WarningMessage object at 0xb1455c90>, 
<warnings.WarningMessage object at 0xb1455fd0>, <warnings.WarningMess...ct at 0xb1455ff0>, 
<warnings.WarningMessage object at 0xb1455e90>, <warnings.WarningMessage object at 0xb1455590>, ...])

wcsaxes/tests/test_misc.py:70: AssertionError
----------------------------- Captured stdout call -----------------------------
{message : RuntimeWarning('invalid value encountered in remainder',), 
 category : 'RuntimeWarning', 
 filename : 'wcsaxes/coordinate_helpers.py', lineno : 28, line : None}
[...]
```
Numpy version is 1.9.2. On x86_86, the identical software configuration succeeds. This actually came out opf a reproducibility test.

**Comment by astrofrog** [comment]
@olebole - this should be fixed, can you confirm?

**Comment by olebole** [comment]
It seens that was already fixed in 0.6 with numpy 1.11.0.
